pyrobosim/pyrobosim/gui/world_canvas.py

```diff
@@ -150,6 +154,46 @@ def toggle_collision_polygons(self) -> None:
         self.show_hallways()
         self.show_rooms()
 
+    def toggle_room_names(self) -> None:
+        """Shows/hides room names."""
+        self.show_room_names = not self.show_room_names
+        self.world.logger.info(
+            "Enabling room names" if self.show_room_names else "Disabling room names"
+        )
```

Comment:
suggestion: You could shorten these by doing something like

```python
("Enabling" if True else "Disabling") + " object names"
```

same holds for the other functions